src/components/app/App.js

```diff
@@ -1,24 +1,28 @@
 import React from 'react';
-import { Header } from '../header/HeaderFromLecture';
+import {Header} from '../header/HeaderFromLecture';
+
+import {Footer} from "../footer/Footer";
 
-// todo: здесь нужно сделать импорт Footer из '../footer/Footer'
 
 // todo: здесь нужно сделать импорт usersList из '../../constants'
+import {usersList} from "../../constants";
 // todo: здесь нужно сделать импорт UserCard из '../user-card/UserCard'
+import {UserCard} from '../user-card/UserCard';
 
 import './App.css';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
-  return (
-    <div className="App">
-      <Header />
-      <div>content will appear here soon</div>
-      {/*  todo: здесь нужно использовать Footer по аналогии с Header */}
+    return (
+        <div className="App">
+            <Header/>
+            <div>content will appear here soon</div>
+            <Footer/>
 
-      {/*  todo: здесь нужно использовать UserCard по аналогии с Header, как пропсу user передать usersList[0]*/}
-    </div>
-  );
+            {/*  todo: здесь нужно использовать UserCard по аналогии с Header, как пропсу user передать usersList[0]*/}
+            <UserCard usersList={usersList[0]}/>
```

Comment:
Let's call this prop just `user` not `usersList `, so it would be `users={usersList[0]}`

And footer should be the last one on the page =)